subprojects/resources-http/build.gradle.kts

```diff
@@ -17,6 +17,12 @@ plugins {
     id("gradlebuild.distribution.api-java")
 }
 
+repositories {
+    maven {
+        url = uri("http://dl.bintray.com/milton/Milton")
+    }
+}
```

Comment:
Why isn't this artifact served from central or JCenter? I'm against adding some third party artifact server to our Gradle build.

Reply:
That's a question for the author of the artifact. I understand your concern here, I'll see if I can find another solution.